### PR TITLE
Fix Issue #120: Fix parallel for <test> (DTD)

### PR DESCRIPTION
**Pull request by juherr**
Fix Issue #120: Fix parallel for <test> (DTD)
+
Replace parallel string values by a Java5 enum

### CODE REVIEW

File: src/main/java/org/testng/xml/TestNGContentHandler.java [hunk 1]
```diff
@@ -154,9 +154,9 @@ private void xmlSuite(boolean start, Attributes attributes) {
       }
       String parallel = attributes.getValue("parallel");
       if (parallel != null) {
-        if (isValidParallel(parallel)) {
```
Comment (cbeust) [inline src/main/java/org/testng/xml/TestNGContentHandler.java]: Maybe this snippet and the next one could be captured in the same method?

**Comment by cbeust** [comment]
LGTM after one comment.

**Comment by juherr** [comment]
@cbeust Done :)

**Comment by cbeust** [comment]
Thanks @juherr !

**Pull request merged**